### ISSUE TITLE
[A11y]Focus is not visible on the expand/collapse button

### ISSUE THREAD

**Issue by ryuyu** [NuGet/NuGetGallery#7885]
Copied from https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1069655
Repro Steps:​
1.Hit the URL "https://www.nuget.org" and sign in with valid credentials​
2.Tab till the user menu button and press enter to expand it. Select "Manage Packages" and press enter​
3.Tab till "Published Packages", expand it and select any existing package.(If no packages are available, add a new package)​
4.Tab till "Manage Package" link under "Info" and press enter​
5.Tab till "Deprecation" section, enter required details, tab till "Save" button and press enter​
6.Tab till the package name in breadcrumb navigation and press enter​
7.Navigate to https://www.nuget.org/packages/DeltaX.fabel/​
8.Tab till the expand button present for the alert message that is displayed below the page heading​
9.Verify if the focus is visible on the Expand/Collapse button​
​
Actual Result:​
Focus is not visible on the expand/collapse button​
​
Expected Result:​
Focus should be clearly visible on the expand/collapse button​

Notes:
We will need to add some stuff in our CSS for the deprecation container focus outline. The current appears to be overridden to be non-existent.

**Comment by ryuyu** [comment]
Looks like we can set outline-style for this element (focused) to "auto" and this fixes the issue.

**Comment by ryuyu** [comment]
This was actually introduced by https://github.com/NuGet/NuGetGallery/pull/7326 when trying to address https://github.com/NuGet/NuGetGallery/issues/7305
